### ISSUE TITLE
wow, 2.8, scenes

### ISSUE THREAD

**Issue by j2l** [philippe-lavoie/blender-fountain-addon#2]
Hi Phil, 
First of all, bravo ! It's a gem you're creating.
I'd love to see it working in Blender 2.8. I don't use 2.79 anymore, well it's still there for your add-on :)

I also tried Blender as a storyboarding tool because I prefer 3D to hand drawing and camera moves are easy to implement into a scene.

Just one thing though: Multiple scenes should be created based on fountain script. In your video, everything is created under the same Blender scene.

**Comment by j2l** [comment]
bump

**Comment by 3dcinetv** [comment]
How possible would it be to port this script to Blender 2.83 and beyond? This is incredible! It solves all of the problems of AV-Sync when scenes are heavy. Specially when combining 2D and 3D animation in Blender. 
Is there a slight hope to see this addon converted to Blender 2.83, please?
Thanks.

**Comment by j2l** [comment]
The author said he will ... last year: https://blenderartists.org/t/fountain-script-to-help-storyboarding/1123522/33
Maybe he can only work on it on August?
Another project is 2.8 but not as featured: https://github.com/tin2tin/Blender_Screenwriter

**Comment by 3dcinetv** [comment]
Thanks for your reply J2L. 
Question regarding the Fountain Script: When I click on "IMPORT" to get the TXT data block into the Fountain addon, what happens when I change the TEXT block in the TEXT editor (say I need to delete some lines), how do I re*send or refresh those changes back to the Fountain addon?
It seems : "UPDATE script" is the only button to do this, but in reality what it does is to update the markers from the timeline to the text block (in the text editor).

**Comment by j2l** [comment]
Don't know, I switched to 2.8 months ago

**Comment by 3dcinetv** [comment]
Indeed. I am working in 2.8 as well. I just wanted to know if the text for the script could be EDITED in the text editor and then (HOW) to propagate this change in dialogue, back in the timeline?
